src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomComponentSerializationService.cs

```diff
@@ -277,7 +281,7 @@ internal void AddMember(object value, MemberDescriptor member, bool absolute)
                     _objectNames.Add(data._name);
                 }
 
-                Trace("Adding object '{0}' ({1}:{2}) {3}", data._name, data._value.GetType().FullName, member.Name, (absolute ? "NORMAL" : "ABSOLUTE"));
+                Debug.WriteLineIf(s_trace.TraceVerbose, $"ComponentSerialization: Adding object '{data._name}' ({data._value.GetType().FullName}:{member.Name}) {(absolute ? "NORMAL" : "ABSOLUTE")}");
```

Comment:
It seems it should be this way:
```suggestion
                Debug.WriteLineIf(s_trace.TraceVerbose, $"ComponentSerialization: Adding object '{data._name}' ({data._value.GetType().FullName}:{member.Name}) {(absolute ? "ABSOLUTE" : "NORMAL")}");
```
Could you please fix it here and in `AddObject`?

Reply:
good catch